src/App.scss

```diff
@@ -73,84 +104,109 @@ body {
     margin-left: 50px;
 }
 
-.App__program-section {
-    margin-top: 1.5rem;
+.App__world-selector-container {
+    width: 6rem;
+    grid-column-start: 4;
+    grid-column-end: 5;
+    grid-row-start: 2;
+    grid-row-end: 3;
 }
 
 .App__command-palette {
     background-color: #F1F1F1;
-    height: 100%;
-    padding-top: 2rem;
     text-align: center;
+    margin: 0 0.5rem;
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 4;
+    overflow: hidden;
 }
 
 .App__command-palette-heading {
     font-weight: bold;
     font-size: inherit;
-    margin: 0 0 2rem 0;
+    margin: 2rem 0;
+}
+
+.App__command-palette-command-container {
+    width: 100%;
+    height: 100%;
+    padding: 1rem;
+    overflow-y: scroll;
 }
 
 .App__command-palette-commands {
-    // width = #items * itemWidth + gapWidth * (#items - 1)
-    width: 14rem;
-    margin: 3rem auto 0 auto;
+    width: 6rem;
     display: grid;
-    grid-template-columns: repeat(3, 4rem);
+    grid-template-columns: repeat(2, 2.5rem);
     grid-gap: 1rem;
 }
 
-.App__playAndShare-container {
+.App__program-block-editor {
+    margin-right: 0.5rem;
+    grid-column-start: 3;
+    grid-column-end: 5;
+    grid-row-start: 3;
+    grid-row-end: 5;
+    height: auto;
+}
+
+.App__playAndShare-background {
+    grid-column-start: 1;
+    grid-column-end: 5;
+    grid-row-start: 4;
+    grid-row-end: 5;
     background-color: #F1F1F1;
-    height: 10rem;
-    margin-top: 1rem;
+    height: 7rem;
+}
+
+.App__playAndShare-container {
```

Comment:
For the same reason as its background, this container's grid-column-end needs to be updated as well.

`grid-column-end: 5`

Reply:
Fixed.